launchpod/angular-launchpod/src/app/my-feeds-page/my-feeds-page.component.ts

```diff
@@ -7,8 +18,43 @@ import { Component, OnInit } from '@angular/core';
 })
 export class MyFeedsPageComponent implements OnInit {
 
-  constructor() {}
+  hasNewFeed: boolean;
+  myFeeds: Feed[];
+
+  constructor(private formHandlerService: FormHandlerService, public snackBar: MatSnackBar) {}
+
+  ngOnInit(): void {
+    this.formHandlerService.myFeeds.subscribe((feeds) => {
+      console.log("Feeds: " + feeds);
+      this.myFeeds = feeds;
+      this.formHandlerService.hasNewFeed.subscribe((result) => {
+        console.log("newFeed my: " + result);
+        this.hasNewFeed = result;
+      });
+    });
+    
+    
+  }
+
+  // Send the key for the feed the user wants to delete to the backend.
+  public deleteFeed(key) {
+    let formData = new HttpParams();
+    formData = formData.set('keyId', key);
+    console.log("Key: " + key);
 
-  ngOnInit(): void {}
+    this.formHandlerService.deleteFeedEntity(formData)
+      .subscribe((feeds) => {
+        console.log("Feeds after delete: " + feeds);
+        this.formHandlerService.sendMyFeeds(feeds);
+        this.hasNewFeed = false;
+        this.myFeeds = feeds;
+      });
+    
+  }
 
+  openSnackBar() {
+    this.snackBar.open('Copied URL to clipboard', '', {
```

Comment:
Nit: what is the second parameter?

Maybe try inline commenting the parameters

> Comments when calling a function
If needed, document parameters at call sites inline using block comments. Also consider named parameters using object literals and destructuring. The exact formatting and placement of the comment is not prescribed.

```
// Inline block comments for parameters that'd be hard to understand:
new Percolator().brew(/* amountLitres= */ 5);
// Also consider using named arguments and destructuring parameters (in brew's declaration):
new Percolator().brew({amountLitres: 5});
```

Reply:
done.